pkg/management/authorization.go

```diff
@@ -265,13 +265,27 @@ func (c *authorizationComponentMW) SetGroupsToUser(ctx context.Context, realmNam
 	}
 
 	action = MGMTAssignableGroupsToUser.String()
-	for _, groupID := range groupIDs {
-		if err := c.authManager.CheckAuthorizationOnTargetGroupID(ctx, action, targetRealm, groupID); err != nil {
-			return err
-		}
+	if err := c.authManager.CheckAuthorizationOnTargetGroupID(ctx, action, targetRealm, groupID); err != nil {
+		return err
+	}
+
+	return c.next.AddGroupToUser(ctx, realmName, userID, groupID)
+}
+
+func (c *authorizationComponentMW) DeleteGroupForUser(ctx context.Context, realmName, userID string, groupID string) error {
+	var action = MGMTSetGroupsToUser.String()
+	var targetRealm = realmName
+
+	if err := c.authManager.CheckAuthorizationOnTargetUser(ctx, action, targetRealm, userID); err != nil {
+		return err
+	}
+
+	action = MGMTAssignableGroupsToUser.String()
```

Comment:
same action as when we add ?

Reply:
Add/Delete methods have MGMTSetGroupsToUser rights to be able to update groups of user and have MGMTAssignableGroupsToUser to know which groups can be added/removed